cypress/plugins/index.js

```diff
@@ -1,17 +1,63 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
-
-// This function is called when a project is opened or re-opened (e.g. due to
-// the project's config changing)
-/* eslint-disable no-unused-vars */
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+const getChallenge = require('../../curriculum/getChallenges');
+const env = require('../../config/env.json');
+const assert = require('assert');
+
+function createPaths(curriculum, superblock, block) {
+  let challengeArr = curriculum[superblock]['blocks'][block]['challenges'];
+
+  return challengeArr.map(
+    challengePath =>
+      `/learn/${superblock}/${block}/${challengePath['dashedName']}`
+  );
+}
+
+module.exports = on => {
+  on('task', {
+    // return an array of all challenge paths
+    //  superBlock, block,  this are unused variables in the condition Husky doesn't like that
+    getCurriculum({ lang }) {
```

Comment:
I want the language to default to English do you agree?

Reply:
Sure, that seems fine.  We can think about using `CURRICULUM_LOCALE` etc. another time. This PR is already doing plenty.

Reply:
i'll leave it for now